### DISCUSSION TITLE
Setting Option to Disable Gutenberg

### DISCUSSION THREAD

**Discussion by SteveFeher** [elementor/elementor#24559] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There are two plugins that I always install first -- Elementor Pro and another called "disable gutenberg".   
Why do I need two plugins?   Why can't we **disable Gutenberg** within the Elementor settings? 

### Describe the solution you'd like

Can we not have a checkbox within the settings of Elementor that disables Gutenberg (probably via .htaccess or wp-config.php, I suppose)?    

### Describe alternatives you've considered

Right now, I need to add in another plugin that's just clumsy and seems unnecessary to the site.   Would love to avoid that second plugin.

**Comment by jazir555** [comment]
Additionally, if the "Disable Gutenberg" function is added, please replace the Gutenberg editor in the backend with the classic editor, just as the Disable Gutenberg plugin does, with the "edit with elementor" button option.
